types/@hmcts/nodejs-logging/Logger.ts

```diff
@@ -0,0 +1,7 @@
+declare module '@hmcts/nodejs-logging' {
```

Comment:
can this be moved to nodejs-logging instead?

Reply:
yup ... i was working on it to get ti to pass first then Ill move it over :D